CGPs/0009.md

```diff
@@ -0,0 +1,81 @@
+
+### Towards a more decentralized reserve
+The Celo community has expressed desire for the reserve to become more decentralized.  One key driver for this is the development of automated cross chain bridges that would allow reserve management to be automated.  Presently, the [Ren project](https://renproject.io/) provides a trustless bridge between Zcash and Ethereum.  Porting this to Celo would likely be a relatively light engineering lift.  A [privacy preserving trustless bridge](https://github.com/ZcashFoundation/zcash-pegzone) is also under construction between Zcash and Cosmos.  Other Zcash<->Ethereum bridges have been proposed by the organizations behind [tBTC](https://tbtc.network/) and [wBTC](https://wbtc.network/).  
+
+Funding for ongoing Zcash development is presently available from the [Zcash Foundation grants program](https://grants.zfnd.org/) and the forthcoming [Zcash Major Grants program](https://www.zfnd.org/blog/major-grants-review-committee-selection-process/).  It's likely that a proposal to build a trustless bridge between Zcash and Celo could be co-funded by sources from both the Zcash and Celo communities.
+
+### Cryptographic DNA
+The Zcash project was founded upon strong science, and the academic work of the Zcash [scientific founders](https://electriccoin.co/team/#scientists) has been foundational to various aspects of cryptocurrency.  Notably, [Eran Tromer](https://www.cs.tau.ac.il/~tromer/), who was one of the Zcash founding scientists, is also an advisor to the Celo project. Tromer has recently contributed to the development of the new [Plumo light client protocol](https://docs.zkproof.org/pages/standards/accepted-workshop3/proposal-plumo_celolightclient.pdf), which leverage zk-SNARKs to enable mobile devices to use Celo without downloading the entire blockchain.
+
+### About the author
+My name is [David Campbell](https://keybase.io/alchemydc), and I am responsible for operations at the ECC.  I also operate (personally) a [validator group](https://www.zanshindojo.org/) on the Celo network, and participated in the Great Celo Stake Off.  I also contributed a [significant refactor](https://github.com/celo-org/celo-monorepo/pull/3888) of the Terraform code published by cLabs to support running mainnet validators in GCP.  More information about my interest in cryptography and privacy and a disclosure statement is available [here](https://forum.zcashcommunity.com/t/dc-for-mgrc/36666).
+
+## Proposed Changes
+The present target allocation of the Celo reserve is as follows:
+* 50% CELO
+* 30% BTC
+* 15% ETH
+* 5% DAI
+
+This proposal advocates that the target allocation of the Celo reserve be modified as follows:
+* 50% CELO
+* 30% BTC
+* 10% ETH
+* 5% DAI
+* 5% ZEC
+
+
+## Verification
+This proposal involves no code changes or verification.  
```

Comment:
It should change `assetAllocationSymbols` I believe, see https://github.com/celo-org/celo-monorepo/blob/8a8b82252e2da9a4afc743b1e27260d521729f74/packages/protocol/migrationsConfig.js#L153